src/pow.cpp

```diff
@@ -73,22 +73,3 @@ unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, int64_t nF
 
     return bnNew.GetCompact();
 }
```

Comment:
@jtimon actually, it is not ok, pow is not part of consensus (some methods are linked to CBlockIndex)